source/nodejs/package-lock.json

```diff
@@ -12759,7 +12759,7 @@
     },
     "tty-browserify": {
       "version": "0.0.0",
-      "resolved": "https://registry.npmjs.org/tty-browserify/-/tty-browserify-0.0.0.tgz",
+      "resolved": "http://registry.npmjs.org/tty-browserify/-/tty-browserify-0.0.0.tgz",
```

Comment:
Nit: that's a funny change it's making to the package-lock file (moving away from https to http)  -- wondering if that's an environment setting of some sort or this npm bug ? Quick hunting around resulted in this [https://discuss.circleci.com/t/npm-install-downgrades-package-lock-json-links-from-https-to-http/22508/7](https://discuss.circleci.com/t/npm-install-downgrades-package-lock-json-links-from-https-to-http/22508/7) ..what's your npm version say ? (npm --version) ?

Reply:
6.9.0

---
In reply to: [315413547](https://github.com/microsoft/AdaptiveCards/pull/3350#discussion_r315413547) [](ancestors = 315413547)

Reply:
I think that should be good -- try this per that last thread -- see if that will get rid of the http changes to package-lock
rm -rf ./node_modules
npm cache clean --force
npm install --prefer-online

---
In reply to: [315415294](https://github.com/microsoft/AdaptiveCards/pull/3350#discussion_r315415294) [](ancestors = 315415294,315413547)

Reply:
Tried that, reverted the package-lock file, and rebuilt everything. I'm not getting any changes to the package-lock file anymore so looks like that fixed me. Thanks.

---
In reply to: [315416611](https://github.com/microsoft/AdaptiveCards/pull/3350#discussion_r315416611) [](ancestors = 315416611,315415294,315413547)